### DISCUSSION TITLE
Welcome to awesome-blockchain-cn Discussions!

### DISCUSSION THREAD

**Discussion by chaozh** [chaozh/awesome-blockchain-cn#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by querqing** [comment]
来信均悉。

**Comment by xx77xx** [comment]
您好，我已经成功收到了邮件!尽快给您回复。

**Comment by lizj3624** [comment]
程序员都可以参与那些入门的区块链项目？

**Reply by bitmasterv** [reply]
我这里有区块链项目是否感兴趣，dily3825002@163.com，如果感兴趣可以联系我，或者加微信 dily1940

**Comment by lghthon** [comment]
您好，请问在国内求职的话需要准备一个什么样的区块链项目？另外，有哪些工作岗位推荐。期盼您的回复！

**Reply by bitmasterv** [reply]
我这里有区块链项目是否感兴趣，dily3825002@163.com，如果感兴趣可以联系我，或者加微信 dily1940
